docs/ECLWatch/TheECLWatchMan.xml

```diff
@@ -190,22 +188,18 @@
           options. When you specify any Filter options, the Filter action
           button changes and then displays <emphasis role="bold">Filter
           Set</emphasis> and the filter indicator image lights up. When you
-          then copy an <emphasis>Open new Page</emphasis> link with the filter
-          set, the filter condition persists. Filter conditions continue to
-          refine the results displayed until you clear the filter.</para>
+          then copy an <emphasis role="bold">Open in New Page</emphasis> link
+          with the filter set, the filter condition persists. Filter
+          conditions limit the results displayed until you clear the
+          filter.</para>
         </sect3>
       </sect2>
 
       <sect2 id="ECLWA_BannerItems">
         <title>Banner Items</title>
 
-        <para>These are the items along the top banner area of the main ECL
-        Watch page. Several of these items are detailed either later in this
-        document or in a separate document, such a the systems monitor
-        indicator light. That is touched on briefly here but for more
-        information refer to the HPCC Monitoring and Reporting doc.</para>
-
-        <para>The Banner Items are a constant in the interface. The top banner
+        <para>The banner area of ECL Watch appears at the top of all pages.
```

Comment:
The banner area of ECL Watch appears at the top of all pages. The Banner Items are a constant in the interface. The top banner remains a fixture at the top of your window as you navigate through the pages.

3 sentences when one will do. Keep "The banner area of ECL Watch appears at the top of all pages."